rl_baselines/base_classes.py

```diff
@@ -136,7 +137,14 @@ def save(self, save_path, _locals=None):
         }
         with open(save_path, "wb") as f:
             pickle.dump(save_param, f)
-
+    def setLoadPath(self, load_path):
+        """
+        Load the only the parameters of the neuro-network model from a path
```

Comment:
```suggestion
        Load the parameters of a trained rl model from a path
```

this method does not do what it says...